### PR TITLE
fix: simple request

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/web3-providers/src/kv/index.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class JSON_Storage implements StorageAPI.Storage {
                 {
                     method: 'GET',
                     headers: {
-                        'Content-Type': 'application/json',
+                        'Content-Type': 'text/plain',
```
Comment (septs) [inline packages/web3-providers/src/kv/index.ts:18]: ？

Comment (limichange) [reply packages/web3-providers/src/kv/index.ts:18]: Interesting.

Comment (Jack-Works) [reply packages/web3-providers/src/kv/index.ts:18]: Can you explain why? @lelenei

Comment (leavesster) [reply packages/web3-providers/src/kv/index.ts:18]: it cant make CORS great again.

Comment (himself65) [reply packages/web3-providers/src/kv/index.ts:18]: 😨

Comment (Jack-Works) [reply packages/web3-providers/src/kv/index.ts:18]: > it cant make CORS great again.

Can you explain in detail? Are we failing on the CORS request? Or do you want to save an OPTIONS request on this?

Comment (lelenei) [reply packages/web3-providers/src/kv/index.ts:18]: I don't send OPTIONS requests

Comment (nuanyang233) [reply packages/web3-providers/src/kv/index.ts:18]: > I don't send OPTIONS requests

Please explain it.

Comment (guanbinrui) [reply packages/web3-providers/src/kv/index.ts:18]: Are you trying to avoid preflight requests?

**Review by guanbinrui — APPROVED** [review]

File: packages/web3-providers/src/kv/index.ts (hunk 1 above)
Comment (lelenei) [reply packages/web3-providers/src/kv/index.ts:18]: yes,
content-type only supports text/plain,multipart/form-data,application/x-www-form-urlencoded

**Pull request merged**